Codeinwp/neve
# New typography controls [Codeinwp/neve-pro-addon#184]

### Summary
Adds new typography customizer controls.


### Will affect visual aspect of the product

NO

### Test instructions


- Check if all the values are migrated properly from earlier versions. 
- Check if the controls work as expected [live refresh included]


Closes Codeinwp/neve-pro-addon#184.
